apps/aecore/lib/aecore/structures/data_tx.ex

```diff
@@ -0,0 +1,88 @@
+defmodule  Aecore.Structures.DataTx do
+  @moduledoc """
+  Aecore structure of a transaction data.
+  """
+
+  alias Aecore.Keys.Worker, as: Keys
+  alias Aeutil.Serialization
+  alias Aeutil.Parser
+
+  @typedoc "Reason for the error"
+  @type reason :: String.t()
+
+  @typedoc "Structure of the main transaction wrapper"
+  @type t :: %__MODULE__{
```

Comment:
why is `%__MODULE__` syntax used here?

Reply:
multiple times in this file

Reply:
Because we are in the specified module and we can use this insted of an alias to itself. I think it's less confusing, if you think otherwise I can change it.

Reply:
I'd prefer to be specific about it, so by reading this line I know what happens, independently from the module